Dockerfile

```diff
@@ -73,7 +87,7 @@ RUN swipl -g "pack_install(sparqlprog, [interactive(false)])" -g halt && \
     ln -sf /root/.local/share/swi-prolog/pack/sparqlprog /tools/
 
 # Install obographviz
-RUN npm install obographviz && \
+RUN npm install -g obographviz && \
     ln -s /tools/node_modules/obographviz/bin/og2dot.js /tools/og2dot.js
```

Comment:
If you go for a “global” install, then you can remove the `ln` command. It’s no longer needed since `og2dot` is now installed in `/usr/bin`, so already in the PATH.

Reply:
Done